api/types.go

```diff
@@ -183,3 +183,11 @@ type OrganizationSubscriptionInfo struct {
 	Usage              *db.OrganizationCounters     `json:"usage"`
 	Plan               *db.Plan                     `json:"plan"`
 }
+
+type SubscriptionCheckout struct {
```

Comment:
Please, add a comment to this struct to give more context to the information that it contains.